redisearch/client.go

```diff
@@ -363,6 +363,19 @@ func (i *Client) Drop() error {
 
 }
 
+// Deletes the index
+func (i *Client) DropIndex(indexName string, deleteDocument bool) error {
```

Comment:
I think the parameter `indexName ` replace with `i.name` may be more appropriate